tests/test_pyi.py

```diff
@@ -27,49 +27,52 @@ def checkFile(self, filename: str, pyi_aware: bool) -> Tuple[int, str, str]:
         stderr_text = proc.stderr.decode('utf8')
         return proc.returncode, stdout_text, stderr_text
 
+    def checkFileOutput(self, filename: str, *, pyi_aware: bool = True,
+                        stdout_lines: Sequence[str] = (), stderr_lines: Sequence[str] = ()) -> None:
+        returncode, stdout, stderr = self.checkFile(filename, pyi_aware=pyi_aware)
+        expected_returncode = 1 if stdout else 0
+        self.assertEqual(returncode, expected_returncode, stdout)
+
+        for (actual, expected_lines) in [(stdout, stdout_lines), (stderr, stderr_lines)]:
+            actual = '\n'.join(
+                line.split('/')[-1] for line in actual.split('\n') if line
+            )
```

Comment:
I meant that this is brittle because:
* the first time we add a slash to a message in some error, this will break;
* if an unmatched message is using a slash, we will get a surprising assertion error.

But as I said, this is low pri. You can leave this for now.